apps/core/src/enums/features.enums.ts

```diff
@@ -25,4 +25,5 @@ export enum Feature {
     KnownIotaEVMCoinTypes = 'known-iota-evm-coin-types',
     IotaNames = 'iota-names',
     ValidatorAddressAlias = 'validator-address-alias',
+    TFIdentity = 'trust-framework-identity',
```

Comment:
I suggest renaming to:
```suggestion
    ExplorerTFIdentity = 'explorer-trust-framework-identity',
```

Reply:
Adopted at: https://github.com/iotaledger/iota/pull/9660/commits/2936425d3ff69b2f25f0bac8e03eee9b192282b1